controllers/matryoshka/internal/kubeapiserver/kubeapiserver.go

```diff
@@ -256,6 +256,10 @@ func (r *Resolver) apiServerVolumeMounts(server *matryoshkav1alpha1.KubeAPIServe
 }
 
 func (r *Resolver) apiServerCommand(server *matryoshkav1alpha1.KubeAPIServer) []string {
+	featureGates := make([]string, 0, len(server.Spec.FeatureGates))
```

Comment:
As `capacity` is optional and derived from the `length` we can just use `make([]string, len(server.Spec.FeatureGates))`

Reply:
Forget about my suggestion! Please use length 0. Sorry for the commotion.